app/routes/routes_helper.py

```diff
@@ -0,0 +1,19 @@
+from flask import jsonify, make_response, abort
+from app.models.planet import Planet
+
+def error_message(message, status_code):
+    abort(make_response(message, status_code))
+
+def validate_planet(id):
```

Comment:
👍 
I'm of two minds as to where this function should live. On the one hand, the only routes you have right now are the planet routes, so having this here makes sense. On the other hand, I sort of expect a file called `routes_helper.py` to have general purpose helpers that aren't related to any particular model. So from that perspective, I'd kind of like to see a `planet_route_helpers.py` that has stuff that's for routes and planet models. On the other (third?) hand, we might need to look up a planet from other routes if we were establishing relationships between multiple model types, so it could be fine here!

Deciding where to put stuff is an extension of deciding how to name things, which is really hard!